### PR TITLE
finished

### PR DESCRIPTION
**Pull request by mheijer**

### CODE REVIEW

File: asst1/asst1.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+def bluesclues (abbr) :
+  f = open('states.txt')
+  answer = {}
+  for line in f :
+    k, v = line.strip().split(',')
+    answer[v.strip()] = k.strip()
+  return answer[abbr]
+
+def bluesbooze (name) :
+  f = open('states.txt')
```
Comment (gcmerz) [inline asst1/asst1.py:51]: how could you have used 'bluesclues' to implement 'bluesbooze?' there's a lot of repeat code here that you probably could have avoided

File: asst1/asst1.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      ret += most.lower()
+    else :
+      ret += c
+  return ret
+
+def sortcat (n, *args) :
+  xs = []
+  str = ""
+  i = n
+  for a in args :
```
Comment (gcmerz) [inline asst1/asst1.py:30]: this code is very C-style! Did you take CS50? Overall very good though -- we'l learn how to write code more "pythonically" throughout the comp (:

File: asst1/asst1.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+def bluesbooze (name) :
+  f = open('states.txt')
+  answer = {}
+  for line in f :
+    k, v = line.strip().split(',')
+    answer[k.strip()] = v.strip()
+  return answer[name]
+
+# tests
```
Comment (gcmerz) [inline asst1/asst1.py:58]: Nice job writing tests!! Love to see it. Tests could afford to be a little more comprehensive though -- are there an edge cases you're missing?
